src/main/java/uk/gov/hmcts/reform/userprofileapi/exception/ForbiddenException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.userprofileapi.exception;
+
+public class ForbiddenException extends RuntimeException {
+
+    public ForbiddenException(String message) {
```

Comment:
Use Single exception. May be create generic UserPRofileFailureException and Pass message Forbidden LD